### PR TITLE
posts: Add GSoC

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <ce.gao@outlook.com>

Related issue #0

## Post info

- Title: Google Summer of Code 申请指南
- Author: gaocegege
- License: default
- Estimate finish time: Already

### CODE REVIEW

**Comment by gaocegege** [comment]
CI fails because of the wrong version of golang. It was fixed in #17

**Comment by at15** [comment]
@gaocegege mind update the PR description? the example content should not be there 😄

**Comment by gaocegege** [comment]
Done

File: source/google_summer_of_code_申请指南.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+1. **只有**学生才可以申请 GSoC。
+1. 一般来说 GSoC 主页需要科学上网才能访问。
+1. 时差问题是申请的时候需要注意的问题，这个需要格外注意，每年都有人错过申请。
+1. 奖金的发放是通过 [Payoneer](https://www.payoneer.com/home/) 发放的，如果是非美元账户，需要支付 4% 左右的换汇费用。
+1. 第一次入选 Mentoring organizations 的组织原则上只有 1 个或者 2 个 slots。
+
+## 结语
+
+这是一篇摸鱼作，希望能够对各位有所帮助。其实大家在选择开源社区的时候可以多问问有经验的人，尽可能选择一个友好的社区作为开始，这样会在开源的路上走的远一点。。
```
Comment (at15) [inline source/google_summer_of_code_申请指南.md:124]: 感觉应该说一些比较友好的社区吧，以及贡献的注意事项 (埋灰的 https://github.com/dyweb/mos/issues/18

Comment (gaocegege) [reply source/google_summer_of_code_申请指南.md:124]: 那要拖一段时间了。。

Comment (at15) [reply source/google_summer_of_code_申请指南.md:124]: 我 LGTM 打到一半发现 title 变成 WIP 了.... 感觉还是先发了吧, WIP 就不知道什么时候了 ....

**Comment by at15** [comment]
LGTM

LGTM

[Approved with PullApprove](https://pullapprove.com/dyweb/blog/pull-request/18/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=dyweb/blog)

File: source/google_summer_of_code_申请指南.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
+title: "Google Summer of Code 申请指南"
+date: 2017-07-11 09:02:34 +0000
+update: 2017-07-11 09:02:34 +0000
+author: gaocegege
+# NOTE: the image locate in `source` folder
+# cover: "-/images/posts/example.png"
+tags:
+    - "Google Summer of Code"
+    - "申请指南"
+preview: "本文的受众主要是想在暑假找点事情做，挣点外块的同学，亦或是想积累一下参与真实软件开发经验的同学。"
```
Comment (arrowrowe) [inline source/google_summer_of_code_申请指南.md]: Maybe [外快](https://zh.wiktionary.org/wiki/%E5%A4%96%E5%BF%AB)?

**Comment by at15** [comment]
cc @arrowrowe @ComMouse need another LGTM from any of you

**Comment by arrowrowe** [comment]
我没啥意见, 上面的错别字改下就好了...

LGTM:sheep:

[Approved with PullApprove](https://pullapprove.com/dyweb/blog/pull-request/18/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=dyweb/blog)

**Review by arrowrowe — APPROVED** [review]

**Comment by at15** [comment]
@gaocegege it seems you need to merge it manually due to CI failing ....

**Review by at15 — APPROVED** [review]
.... e, dual approve

**Comment by gaocegege** [comment]
OK

**Comment by gaocegege** [comment]
remote: error: GH006: Protected branch update failed for refs/heads/master.
remote: error: Required status check "code-review/pullapprove" is expected.
To github.com:dyweb/blog.git
 ! [remote rejected] master -> master (protected branch hook declined)
error: failed to push some refs to 'git@github.com:dyweb/blog.git'

**Comment by gaocegege** [comment]
=。= 原来改标题就好了

**Comment by gaocegege** [comment]
修改了 typo，要重新 review。。

**Comment by at15** [comment]
LGTM  /w\

[Approved with PullApprove](https://pullapprove.com/dyweb/blog/pull-request/18/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=dyweb/blog)

**Comment by ComMouse** [comment]
LGTM

**Review by ComMouse — APPROVED** [review]

**Comment by ComMouse** [comment]
LGTM

[Approved with PullApprove](https://pullapprove.com/dyweb/blog/pull-request/18/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=dyweb/blog)

**Comment by at15** [comment]
LGMT

**Comment by at15** [comment]
LGTM

[Approved with PullApprove](https://pullapprove.com/dyweb/blog/pull-request/18/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=dyweb/blog)

**Comment by at15** [comment]
oops, forgot to build the html .... @gaocegege

**Pull request merged**